test_conformance/common/vulkan_wrapper/opencl_vulkan_wrapper.cpp

```diff
@@ -886,60 +983,7 @@ int clExternalSemaphore::signal(cl_command_queue cmd_queue)
     return err;
 }
 
-int clExternalSemaphore::wait(cl_command_queue cmd_queue)
-{
```

Comment:
Any reason we need to remove wait and signal operations on base class clExternalSemaphore?

Reply:
clExternalSemaphore is an abstract class, wait and signal need to be implemented in clExternalExportableSemaphore or clExternalImportableSemaphore.  The class is called c**lExternal**Semaphore, meaning it is used for communicating with external APIs.  There are only two ways to do that, import or export.  You could implement a semaphore that does neither import or export, but putting that functionality in a class named clExternalSemaphore doesn't make sense.  The spec prohibit simultaneous import/export, which is why clExternalSemaphore got split into two classes.  It improves readability and simplifies the constructors, which would otherwise need to handle both import and export.